### DISCUSSION TITLE
Support for border-image-* properties

### DISCUSSION THREAD

**Discussion by MarkusWendorf** [tailwindlabs/tailwindcss#11009] [Ideas]
The border-image property can be quite useful but it seems to be missing in Tailwind.

border-image
border-image-source
border-image-slice
border-image-width
border-image-outset
border-image-repeat

**Comment by sebagnz** [comment]
I created a [plugin to add utilities for border-image](https://github.com/sebagnz/tailwindcss-border-image) and all its properties. Hope it's useful 😁.
